crates/types/src/data.rs

```diff
@@ -88,6 +88,12 @@ impl ConsensusTime for ViewNumber {
     }
 }
 
+impl Display for ViewNumber {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
```

Comment:
not inherently against this, but is there a reason not to `#[derive(Display)]`?

Reply:
Oh, it doesn't have a `Display` derive

Reply:
oh oops, my bad -- yeah you're right, nevermind: https://doc.rust-lang.org/std/fmt/trait.Display.html